test/fixtures/.shout/config.js

```diff
@@ -0,0 +1,4 @@
+var config = module.exports = require('../../../defaults/config.js');
```

Comment:
Yeah, I have seen that on occasions (like, two), but it doesn't mean it's not ugly. I'd really go for something more explicit instead, which is how it's being used most of the time.